resources/Components/Cluster/index.js

```diff
@@ -1,5 +1,3 @@
-import React from 'react'
```

Comment:
for my own information, what rule causes us to remove these React import statements from lots of files, why aren't they needed, and were they at some point in the past?

Reply:
These were previously required because JSX usage required a React import to be in scope in order to use it.  React 17 changed that with the new JSX transform.

https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md